concepts/functions/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for this concept",
+  "blurb": "A function is a block of code which only runs when it is called. It can pass data, known as parameters, into a function. A function can return data as a result for the further use.",
```

Comment:
> It can pass data, known as parameters, into a function. 

"It" here would be "functions" so this reads "functions can pass data into a function". Which is confusing and circular.